NACT/attention_query_utilities/attention_query.py

```diff
@@ -441,85 +441,89 @@ def set_scanpy_object(self, new_data):
         self.data = new_data
 
 
-# TODO: @Oscar: could you please take care of the clean up and docstrin of this
-# method please?
+# TODO: @Ali: could you take a look at this and see if there are any changes needed. 
 
-    def gene_tfidf(self, top_n_genestrings_dict: dict = None):
-        """Calculate TF-IDF values for all genes in a top-n ranked dictionary.
+    def calculate_tfidf(top_n_names, top_n_df_dict, n_genes=25):
+        """
+        This function receives a pandas series that contains a list of genes in each row, and returns a pandas series with the Corpus-Wide tf-idf scores for each gene.
```

Comment:
**R**: Do not exceed 80 characters per line (see [style guide](https://google.github.io/styleguide/pyguide.html#32-line-length)). The issue happens on multiple lines but I only raise it once here : )

Reply:
@dr-aheydari Not sure if I properly fixed it but this seems similar to this. https://github.com/SindiLab/N-ACT/pull/6#discussion_r1084763239 Should I not have a long description and just make it one line?